src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="get"
+      data-qa="big"
+    >
+      <label>
+        <img
+          src="images/Search.svg"
+          alt="icon"
+          class="image"
+        >
+        <input
+          type="search"
+          name="bigSearch"
+          data-qa="keypress"
+          placeholder="Try &quot;Los Angeles&quot;"
+          class="search-bar"
+        >
+      </label>
+    </form>
+
+    <form
+      action="#"
+      method="get"
+      data-qa="small"
+    >
+      <img
+        src="images/Search.svg"
+        alt="icon"
+        class="image small-image"
```

Comment:
Due to the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)

![image](https://user-images.githubusercontent.com/80700247/230782852-b3ebe27d-05e9-4d9b-9ee0-b03ea96dcc09.png)
```suggestion
        class="image image--small"
```

Fix all such cases